pubspec.yaml

```diff
@@ -12,7 +12,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  dart_vlc_ffi: ^0.1.2
+  dart_vlc_ffi:
+    path: ./ffi
```

Comment:
This is to use current changes. Must be changed back to pub.dev when publishing.